tensorflow_data_validation/utils/display_util.py

```diff
@@ -180,6 +180,8 @@ def visualize_statistics(
   Raises:
     TypeError: If the input argument is not of the expected type.
     ValueError: If the input statistics protos does not have only one dataset.
+
+  Returns: HTML to be embedded for visualization
```

Comment:
To help us keep our style consistent across TFDV, please make the following tweaks to the docstring:

- Move the Returns section above the Raises section
- Add a newline after the colon in "Returns:"
- Add a period after "visualization"

Reply:
I've fixed the docstring and also added tests. PTAL.